frontend/src/console/components/ThemePreview/styles.scss

```diff
@@ -0,0 +1,68 @@
+@import '~styles/theme';
+
+div.preview-header {
+  display: flex;
+  align-items:center;
+  position:relative;
+  height: 60px;
+  z-index: 2;
+
+  div.notice-container {
+    display: flex;
+    flex-direction: row-reverse;
+    width: 80%;
+    height: 100%;
+    margin-bottom: 24px;
+
+    p {
+      color: $primary-1;
+      margin-top: 25px;
+      margin-bottom: auto;
+      font-size: 14px;
+      font-weight: normal;
+      font-stretch: normal;
+      font-style: $chivo-font;
+      line-height: normal;
+      letter-spacing: normal;
+      text-align: right;
+    }
+  }
+}
+
+.preview-box {
+  height: auto !important;
+  width: 100% !important;
+}
+
+.preview-container {
+  z-index: 1;
+  position: relative;
+  padding: 0;
+  .row {
+    padding: 0;
+
+    .side-buttons {
+      padding: 0 15px 0 0;
+
+      .row {
+        display: flex;
+        align-items: center;
+        padding: 0 15px;
+
+        .input-field-color.form-control[readonly] {
+          width: 75%;
+          background-color: white;
+        }
+
+        .input-field-color.form-control[disabled] {
+          background-color: #e9ecef !important;
+        }
+
+        .info-icon {
+          padding: 5px 10px;
+          color: $primary-1;
+        }
+      }
+    }
+  }
+}
```

Comment:
Comments for this are the same as for another styles file. Please check Bootstrap documentation and try to reuse already existing styles. Also, please avoid very complicated selectors like `input-field-color.form-control[disabled]`.

Reply:
I'll apply the suggestions/

Reply:
Most of these were from the old Color component which is to be added in a different task.
I'll delete them; confirm if this is satisfactory.

Reply:
See https://github.com/open-craft/opencraft/blob/08bb149fe91f2d182432848a92c6682280b1920d/frontend/src/console/components/ThemePreviewAndColors/styles.scss#L20